cmd/easeprobe/report.go

```diff
@@ -116,4 +117,7 @@ func scheduleSLA(probers []probe.Prober) {
 	}
 
 	cron.StartAsync()
+
+	_, t := cron.NextRun()
+	log.Infof("The SLA report will be schedule at %s", t.Format(conf.Get().Settings.TimeFormat))
```

Comment:
Do we need this back ? I removed it on the previous PR as it was displaying the same message 2 times. One was from this line and the other from line 94

Reply:
Sorry, I misunderstood at that time. 

This  different is 
- here, the log.Infof() is for the first time when EaseProbe starts. 
- line 94 will be executed after the SLA  report is sent. 

So, without this log, we cannot see when the first SLA report will be sent.